daemon/volumes.go

```diff
@@ -119,7 +119,15 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 	}
 
-	// 3. Read bind mounts
+	// 3. Read tmpfs
+	for destination := range hostConfig.Tmpfs {
+		mountPoints[destination] = &volume.MountPoint{
+			Source:      "tmpfs",
```

Comment:
definitely - I did this because it's already "Source == tmpfs" in `container.Mount` https://github.com/docker/docker/pull/23301/files#diff-ea22bb26655e758dc94c291dbaee0e76R482

I don't think this is blocking though - we can change this after #22373 to be Type == tmpfs (which I like more)